tests/ui/pages/desktop/base.py

```diff
@@ -26,6 +31,16 @@ def login(self, email, password):
     def logout(self):
         self.header.click_logout()
 
+    def wait_for_page_to_load(self):
+        if 'PYTEST_BASE_URL' in os.environ:
+            self.wait.until(
+                lambda _: self.find_element(
+                    *self._promo_box_locator).size['height'] == 271)
+        else:
+            self.wait.until(lambda _: self.find_element(
+                *self._amo_header).is_displayed())
```

Comment:
So does this mean the promo box is not displayed when using a live server? If so, we probably can't use this wait at all, as we should avoid logic in page objects.